code-submissions-clustering-core/src/main/kotlin/org/jetbrains/research/code/submissions/clustering/load/clustering/hac/parallel/ParallelUtils.kt

```diff
@@ -0,0 +1,11 @@
+package org.jetbrains.research.code.submissions.clustering.load.clustering.hac.parallel
+
+object ParallelUtils {
```

Comment:
Should we move into 'org.jetbrains.research.code.submissions.clustering.util.parallel'? Because it is not only about HAC